lib/components/Sequencer/index.jsx

```diff
@@ -122,6 +104,29 @@ export class Sequencer extends Component {
     this.setState({ trackRacks: newRack })
   }
 
+  addTrack(newSound) {
+    const soundFromDB = this.props.sound.library.find((sound)=>{
+      const soundValue = JSON.parse(sound.attributes);
+      if(soundValue.soundName === newSound) {
+        return true
+      }
+    })
+
+    const rightSound = JSON.parse(soundFromDB.attributes)
```

Comment:
@DanGrund What does `rightSound` mean exactly? Don't see a leftSound? Naming is a little confusing but looks good otherwise.